sync-template.sh

```diff
@@ -429,6 +501,25 @@ CONTACTS="${CONTACTS:-${CONTACTS_DEFAULT}}"
 export GITHUB_TOKEN="${GITHUB_TOKEN:-}"
 REPOLIST="${REPOLIST:-}"
 REPOLIST="${REPOLIST//,/ }"
+if [[ $USE_HUB = true ]]; then
+  case $FROM_REPO in
+      http*://github.com/*) FROM_REPO_HUB=${FROM_REPO#http*://github.com/} ;;
```

Comment:
Does it need to support plain `http://` - I hope everything is using https and if not it should be fixed.

Reply:
the use of `http://` is now an error